src/FormBuilderBundle/Resources/config/types/field_types.yml

```diff
@@ -36,6 +36,12 @@ form_builder:
                         type: tagfield
                         label: 'form_builder_type_field.date_years'
                         config: ~
+                    options.html5:
+                        display_group_id: attributes
+                        type: checkbox
+                        label: 'use HTML5 type=date (do not use with class="custom-datepicker")'
```

Comment:
Please use translation keys (eg `form_builder_type_field.date_html5`) and move the related translation to `admin.csv`.